### ISSUE TITLE
App doesn't load translation getters on load

### ISSUE THREAD

**Issue by vinayakmalviya** [vinayakmalviya/Teledermatology#5]
App is not able to load the translation getters in the I18n config. It works only after force updating the application.

Steps to reproduce:
1. `cd <repo>`
2. `react-native start`
3. `react-native run-ios`

Screenshots:
Initial | After refresh
------------ | ------------
![Simulator Screen Shot - iPhone 11 - 2020-05-28 at 22 11 23](https://user-images.githubusercontent.com/42168249/83174025-5fc15380-a137-11ea-8a4e-e1c3fcc0107e.png) | ![Simulator Screen Shot - iPhone 11 - 2020-05-28 at 22 27 46](https://user-images.githubusercontent.com/42168249/83174049-6bad1580-a137-11ea-9c27-4de4f40cf765.png)

One work around for fixing it (need to find a better solution) is as follows:
```javascript
import useForceUpdate from '../hooks/useForceUpdate';

const Login = ({ navigation }) => {
  ...

  const forceUpdate = useForceUpdate();

  useEffect(() => {
    forceUpdate();
  }, []);
  
  ...

}
```

**Comment by dvjn** [comment]
Happens on both iOS and Android.

**Comment by vinayakmalviya** [comment]
Didn't happen on Android device while testing. Happened only on iOS simulator. Will check again

**Comment by vinayakmalviya** [comment]
Can confirm..doesn't happen on Android. Could not reproduce on OnePlus 5, 7 and Huawei Honor 6x

**Comment by dvjn** [comment]
Already addressed in #7
